io/net/bonding.py

```diff
@@ -355,16 +355,15 @@ def test_run(self):
         self.bond_fail(self.mode)
         self.log.info("Mode %s OK", self.mode)
         self.error_check()
+        time.sleep(5)
```

Comment:
why sleep ?

Reply:
it taking few sec to come up the interface, otherwise ssh is failing for next test.

Reply:
interface_wait(). If that did not work, add a comment

Reply:
interface_wait() is not working.